nucypher/crypto/powers.py

```diff
@@ -460,6 +465,8 @@ def _get_static_secret_from_ritual_id(self, ritual_id: int) -> SessionStaticSecr
     def get_pubkey_from_ritual_id(self, ritual_id: int) -> SessionStaticKey:
```

Comment:
Let's rename `_get_static_secret_from_ritual_id` and `get_pubkey_from_ritual_id` to something more generic since signing uses `cohort_id` instead of `ritual_id`

Something like `_get_static_secret_from_id`/`get_pubkey_from_id` or ...?